Documentation/ceph-osd-mgmt.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Ceph OSD Management
+weight: 10610
+indent: true
+---
+
+# Ceph OSD Management
+
+Ceph Object Storage Daemons (OSDs) are the heart and soul of the Ceph storage platform.
+Each OSD manages a local device and together they provide the distributed storage. Rook will automate creation and management of OSDs to hide the complexity
+based on the desired state in the CephCluster CR as much as possible. This guide will walk through some of the scenarios
+to configure OSDs where more configuration may be required.
+
+### OSD Health
+
+The [rook-ceph-tools pod](./ceph-toolbox.md) provides a simple environment to run Ceph tools. The `ceph` commands
+mentioned in this document should be run from the toolbox.
+
+Once the is created, connect to the pod to execute the `ceph` commands to analyze the health of the cluster,
+in particular the OSDs and placement groups (PGs). Some common commands to analyze OSDs include:
+```
+ceph status
+ceph osd tree
+ceph osd status
+ceph osd df
+ceph osd utilization
+```
+
+```console
+kubectl -n rook-ceph exec -it $(kubectl -n rook-ceph get pod -l "app=rook-ceph-tools" -o jsonpath='{.items[0].metadata.name}') bash
+```
+
+## Add an OSD
+
+The [QuickStart Guide](ceph-quickstart.md) will provide the basic steps to create a cluster and start some OSDs. For more details on the OSD
+settings also see the [Cluster CRD](ceph-cluster-crd.md) documentation. If you are not seeing OSDs created, see the [Ceph Troubleshooting Guide](ceph-common-issues.md).
+
+To add more OSDs, Rook will automatically watch for new nodes and devices being added to your cluster.
+If they match the filters or other settings in the `storage` section of the cluster CR, the operator
+will create new OSDs.
+
+## Add an OSD on a PVC
+
+In more dynamic environments where storage can be dynamically provisioned with a raw block storage provider, the OSDs can be backed
+by PVCs. See the `storageClassDeviceSets` documentation in the [Cluster CRD](ceph-cluster-crd.md#storage-class-device-sets) topic.
+
+To add more OSDs, you can either increase the `count` of the OSDs in an existing device set or you can
+add more device sets to the cluster CR. The operator will then automatically create new OSDs according
+to the updated cluster CR.
+
+## Remove an OSD
+
+Removal of OSDs is intentionally not automated. Rook's charter is to keep your data safe, not to delete it. If you are
+really sure you want or need to remove OSDs, it can be done. We just want you to be in control of this action.
+
+To remove an OSD due to a failed disk or other re-configuration, first you should verify that your cluster is not full and that
```

Comment:
I think it's important to mention that a ratio must be respected. For instance: "Make sure you will have enough space on your cluster after removing your OSDs to properly  handle the deletion."

Also something like: "make sure not to remove too many OSDs at once. Make sure to wait for rebalancing in-between".